internal/state/enemy_test.go

```diff
@@ -0,0 +1,31 @@
+package state
+
+import "testing"
+
+func TestEnemyHasDefaultLocation(t *testing.T) {
+  enemy := new(Enemy)
+
+  gotx := enemy.location.x
+  goty := enemy.location.y
+
+  if (gotx != 0.0) {
+    t.Errorf("Initialized enemy did not have default x coord 0.0")
+  }
+  if (goty != 0.0) {
+    t.Errorf("Initialized enemy did not have default y coord 0.0")
+  }
+}
+
+func TestEnemyHasDefaultVelocity(t *testing.T) {
+  enemy := new(Enemy)
+
+  gotx := enemy.velocity.x
+  goty := enemy.velocity.y
+
+  if (gotx != 0.0) {
+    t.Errorf("Initialized enemy did not have default x velocity 0.0")
+  }
+  if (goty != 0.0) {
+    t.Errorf("Initialized enemy did not have default y velocity 0.0")
+  }
+}
```

Comment:
Default geometry test? Also for your `players` tests.

On a related note too, should the default radius be 1 instead of the built in value of 0? It seems weird to have an entity with no size.

Reply:
Default geo test is a good idea, i will add 

agreed that the default radius should be 1, but Go does not have a way to set a default for a struct member that isn't the same as that native type's default.  There's no constructor or anything either. 

To get non-0 defaults we will need to establish a pattern of our own "constructor" methods and make a habit of calling them.  From what I've read, Init() is a common name for struct initializers.   I think that's out of the scope of this PR, but would go well with the next one containing default boundaries